src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -710,13 +710,13 @@ export default class GeFormRenderer extends LightningElement{
             try {
                 this.loadingText = this.CUSTOM_LABELS.geAuthorizingCreditCard;
     
-                const currentCaptureGroup = new ElevateCaptureGroup(this.latestCaptureGroupId);
-                const authorizedGift = await currentCaptureGroup.add(tokenizedGift);
+                const currentElevateBatch = new ElevateBatch(this.latestElevateBatchId);
```

Comment:
We are no longer passing an Id value into the constructor for ElevateCaptureGroup/ElevateBatch as per #6404, which has yet to be merged. Just calling it out here in the event that there are merge conflicts and also, to ensure that updated logic doesn't get inadvertently overwritten.